woof-code/rootfs-packages/ptheme/usr/sbin/ptheme_gtk

```diff
@@ -180,7 +180,12 @@ while [ $# != 0 ]; do
 		case $1 in
 			-t) export THEME="$2"
 				[ ! "$THEME" ] && export THEME="$ACTIVE_THEME"
-				shift;;
+				shift
+				export EXIT_WITHOUT_GUI=1;;
```

Comment:
No need for `export`, if I understand correctly a variable (= not an environment variable) should work just fine